components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/DBScriptRunnerUtil.java

```diff
@@ -124,7 +124,7 @@ private static void executeSQL(String sql, Connection connection) {
 
             String q = "select index_name,index_type,status,domidx_status,domidx_opstatus from user_indexes "
                     + "where index_type like '%DOMAIN%' and (domidx_status <> 'VALID' or domidx_opstatus <> 'VALID')";
-            try (Statement statement = connection.createStatement(); ResultSet rs = statement.executeQuery(q);) {
+            try (Statement statement = connection.createStatement(); ResultSet rs = statement.executeQuery(q)) {
                 while (rs.next()) {
                     if (rs.getString("index_name").equals("API_INDEX")) { // re build index if it has failed.
```

Comment:
Do the equals other way around to avoid null pointer exception. i.e "API_INDEX".equals(rs.getString("index_name")). Check equals operations in other places as well.